pages/api-reference/[...slug].tsx

```diff
@@ -0,0 +1,122 @@
+import { readOpenApiSpec, readStainlessSpec } from "@/lib/openApiSpec";
+import { OpenAPIV3 } from "@scalar/openapi-types";
+import { SidebarSubsection } from "@/data/types";
+import { CONTENT_DIR } from "@/lib/content.server";
+
+import remarkGfm from "remark-gfm";
+import { serialize } from "next-mdx-remote/serialize";
+import rehypeMdxCodeProps from "rehype-mdx-code-props";
+import fs from "fs";
+import { MDXRemote } from "next-mdx-remote";
+import { MDX_COMPONENTS } from "../[...slug]";
```

Comment:
If you merged in `cb-tgraph` branch, you can:
```suggestion
import { MDX_COMPONENTS } from "@/lib/mdxComponents";
```